examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -1073,6 +1189,16 @@ int main(int argc, char* argv[])
         {
             if (myid == 0)
             {
+                if (use_eqp && window_ids && current_window < n_windows
+                        && ti == window_ids->item(current_window))
+                {
+                    // Load eqp and reinitialize ROM operator
+                    cout << "Time window start at " << ti << endl;
+                    get_EQPsol(current_window, load_eqpsol);
+                    romop->SetEQP(load_eqpsol);
+                    ode_solver->Init(*romop);
+                    current_window += 1;
```

Comment:
`++`

Reply:
Out of curiousity, is there a benefit to ++ other than that it looks better?

Reply:
I doubt it, and I guess the compiler generates the same result either way. I think it's just more convenient and so commonly used that `+= 1` makes me stop and think why it's not `++`.